docs/content/stable/yugabyte-platform/back-up-restore-universes/configure-backup-storage.md

```diff
@@ -77,6 +77,12 @@ The following S3 IAM permissions are required:
 "s3:GetBucketLocation"
 ```
 
+### Specify signing region
+
+If you are backing up an on-premises universe using a backup configuration for an S3 storage bucket located in a region that is different than the region where your YugabyteDB Anywhere instance is running, or YugabyteDB Anywhere is not running on AWS, and encounter an error similar to `Cannot list objects in backup location`, you may need to specify a signing region for the S3 backup configuration.
```

Comment:
Not on-premises universe, that should be fine. The important case is if (1) using a non-standard S3 endpoint like a private vpc endpoint or an S3 compatible storage, (2) YBA is running on a different region than the storage bucket when a non-standard endpoint is being used.